test/storage/file.js

```diff
@@ -1025,6 +1025,22 @@ describe('File', function() {
     });
   });
 
+  describe('makePublic', function() {
+    it('should make the file public', function(done) {
+
+      // If this mocked method (THAT DOES NOTHING!) is removed, nothing works.
+      file.acl.makeAclObject_ = function() {};
```

Comment:
@stephenplusplus Maybe you can explain this. If I take this line out, this test does not pass. So much is being mocked, it's hard to follow where things are getting overwritten and why this is required or what it's doing!